src/workerd/util/sqlite.c++

```diff
@@ -9,28 +9,29 @@
 #include <sys/stat.h>
 #include <sqlite3.h>
 #include <kj/vector.h>
+#include <workerd/jsg/jsg.h>
 
 namespace workerd {
 
 #define SQLITE_CALL_NODB(code, ...) do { \
     int _ec = code; \
-    KJ_ASSERT(_ec == SQLITE_OK, "SQLite failed: " #code, sqlite3_errstr(_ec), ##__VA_ARGS__); \
+    JSG_ASSERT(_ec == SQLITE_OK, Error, "SQLite failed: " #code, " ", sqlite3_errstr(_ec), ##__VA_ARGS__); \
```

Comment:
Nit: `JSG_REQUIRE` instead of `JSG_ASSERT`

Reply:
Super-nitpicky but I don't agree, many of the uses of SQLITE_CALL are actually asserts. I guess we'd have to create separate SQLITE_ASSERT and SQLITE_REQUIRE macros.

However, given that ASSERT and REQUIRE are actually the same thing under the hood, probably not worth the effort.